### ISSUE TITLE
Rewind/Forward on Replays

### ISSUE THREAD

**Issue by PTF94** [tekbec/plugin.video.nhl66#6]
Rewind and Forward on Replays practically doesn't work. I can try to skip commercials, but it always ends up getting back to beginning of the video.

**Comment by tekbec** [comment]
This seems to be an issue with NHL.TV streams only. Can you confirm it?

**Comment by PTF94** [comment]
Yes. I ran a VPN to try ESPN, and they work fine. The NHL.TV are a bit buggy.

**Comment by tekbec** [comment]
Not sure if there is anything I can do about that. Kodi have a hard time with NHL.TV streams since the beginning, something about the timestamps used in the manifest seems to confuse the player. Will try to elevate that issue to inputstream.adaptative and see if they come up with a solution.

**Comment by tekbec** [comment]
Replays and rewind should now work on NHL.TV streams with latest release v1.0.11. Can you let me know if that works on your side?

**Comment by theboyk** [comment]
Just updated and tested on my end and can fast forward and rewind NHL.TV feeds. Awesome! Thanks!
